cpp/src/strings/find_multiple.cu

```diff
@@ -38,11 +37,9 @@ std::unique_ptr<column> find_multiple( strings_column_view const& strings,
 {
     auto strings_count = strings.size();
     if( strings_count==0 )
-        return make_empty_strings_column(mr,stream);
+        return make_empty_column(data_type{INT32});
     auto targets_count = targets.size();
-    if( targets_count==0 )
-        return std::make_unique<column>(strings.parent());
-
+    CUDF_EXPECTS( targets_count > 0, "Must include at least one search target" );
```

Comment:
Usually empty inputs --> empty results, no error. So seems like this should also return no error.

Reply:
Yes, empty strings returns an empty results. This is the search targets. The result needs to be at least the size of the strings column but there is no valid data to return. Nulls would not be correct since this used for null rows. And >=0 is not correct because these are valid position values. And <0 indicates not found which is not correct because there is no search to perform. 
So this is basically and invalid condition and therefore an exception.

Reply:
I will not be surprised when Dask files an issue because they expect to get no error when they give an empty search target array...

Reply:
I don't see why nulls would not be correct.